File: Revisão P1/RELACIONAIS/relacionais.cs
using System;

public class Program
{
    public static void Main(string[] args)
    {
        int X = 15;
        int Y = 10;

        Console.WriteLine("--- Operadores Relacionais em C# ---");
        Console.WriteLine($"Valores: X = {X}, Y = {Y}\n");

        // Igual a (==)
        Console.WriteLine($"X == Y (15 == 10)? Resultado: {X == Y}");

        // Diferente de (!=)
        Console.WriteLine($"X != Y (15 != 10)? Resultado: {X != Y}");

        // Maior que (>)
        Console.WriteLine($"X > Y (15 > 10)? Resultado: {X > Y}");

        // Menor que (<)
        Console.WriteLine($"X < Y (15 < 10)? Resultado: {X < Y}");

        // Maior ou igual a (>=)
        Console.WriteLine($"X >= Y (15 >= 10)? Resultado: {X >= Y}");
        
        // Menor ou igual a (<=)
        Console.WriteLine($"X <= Y (15 <= 10)? Resultado: {X <= Y}");
    }
}

File: Revisão P1/RELACIONAIS/relacionais.cpp
#include <iostream>
#include <iomanip> // Para std::boolalpha

int main() {
    int X = 15;
    int Y = 10;

    std::cout << "--- Operadores Relacionais em C++ ---\n";
    std::cout << "Valores: X = " << X << ", Y = " << Y << "\n\n";

    // O manipulador std::boolalpha faz com que o resultado seja True/False
    std::cout << std::boolalpha; 
    
    // Igual a (==)
    std::cout << "X == Y (15 == 10)? Resultado: " << (X == Y) << "\n";

    // Diferente de (!=)
    std::cout << "X != Y (15 != 10)? Resultado: " << (X != Y) << "\n";

    // Maior que (>)
    std::cout << "X > Y (15 > 10)? Resultado: " << (X > Y) << "\n";

    // Menor que (<)
    std::cout << "X < Y (15 < 10)? Resultado: " << (X < Y) << "\n";

    // Maior ou igual a (>=)
    std::cout << "X >= Y (15 >= 10)? Resultado: " << (X >= Y) << "\n";
    
    // Menor ou igual a (<=)
    std::cout << "X <= Y (15 <= 10)? Resultado: " << (X <= Y) << "\n";

    return 0;
}